experiences/featured_article_amp/featured_article_amp.test.js

```diff
@@ -0,0 +1,91 @@
+const { bootstrapExperience } = require('../../e2e/utils/bootstrap');
+const { scrollTo } = require('../../e2e/utils/scroll');
+const { touchCard } = require('../../e2e/utils/touch');
+const { isAtSnapPoint } = require('../../e2e/utils/snapPoints');
+const { isCardContentLoaded } = require('../../e2e/utils/cardContent');
+const { expect } = require('chai');
+const { getUrlFixture } = require('../../e2e/utils/fixtureUrl');
+const experience = require('./featured_article_amp.json');
+
+describe('featured_article_amp experience', function() {
+	let config,
+		fixture;
+	const fixtureUrl = getUrlFixture({
+		siteUrl: 'https://playground.marfeel.com/templates/article-skeleton.html',
+		requestHostname: 'playground.marfeel.com',
+		technology: 'web',
```

Comment:
I guess we need the amp version for this test right?

Reply:
> I guess we need the amp version for this test right?

Yes. Applied. it fails. Moving it to draft PR and adding label